pkg/mimirtool/commands/remote_read.go

```diff
@@ -111,6 +113,10 @@ func (c *RemoteReadCommand) Register(app *kingpin.Application, envVars EnvVarNam
 	exportCmd.Flag("tsdb-path", "Path to the folder where to store the TSDB blocks, if not set a new directory in $TEMP is created.").
 		Default("").
 		StringVar(&c.tsdbPath)
+
+	dumpCmd.Flag("chunk-digest", "Print chunk digest information (min time, max time, checksum) instead of decoding samples.").
```

Comment:
```suggestion
	dumpCmd.Flag("chunk-digest", "Print chunk metadata (min time, max time, checksum) instead of decoding samples when using chunked responses. Can only be combined with --use-chunks").
```